civihr_employee_portal/src/Forms/OnboardingWizardCustomizationForm.php

```diff
@@ -45,6 +47,12 @@ public function onSubmit($form, &$formState) {
 
     $this->saveLogo($formState);
     $this->updateCarouselContent($formState);
+
+    // sends user to the first step of the onboarding wizard
+    // which is just node edit page with custom theming
+    if ($this->clickedSaveAndTest($formState)) {
+      drupal_goto('/user/'. user_uid_optional_load()->uid .'/edit', ['query' => ['testing'=>1]] );
```

Comment:
Please add spaces between all the operators and operands here.

Reply:
Right, thanks, I will start using PHP-CS for that kind of formatting.

Reply:
@powdevel Buildkit already includes this tool called [civilint](https://docs.civicrm.org/dev/en/latest/tools/civilint/), which is just PHP-CS configured for the code standards used in CiviCRM. You might want to have a look at it

Reply:
Thanks, good advice. I will check that tool.